index.html

```diff
@@ -11,21 +11,21 @@
     <header>
       <div class="container">
         <div class="row">
-          <div class="col-6">
-              <h1>Student Grade Table</h1>
+          <div class="col-xl-8 text-md-left text-center">
+            <h1>Student Grade Table</h1>
           </div>
-          <div class="col-6">
-              <h2 class="text-right">Average Grade<span class="badge badge-secondary"></span></h2>
+          <div class="col-xl-4 text-md-right text-center">
+            <h2>Average Grade<span class="badge badge-secondary"></span></h2>
           </div>
       </div>
     </header>
     <div class="container">
       <div class="row">
-        <div class="col-8">
+        <div class="col-xl-8">
```

Comment:
When using a `col-*` class that's only for a sub-set of device sizes (`sm`, `md`, `lg`, `xl`), make sure to provide a fallback `col` class for the smaller device sizes.